### ISSUE TITLE
verify_client_object

### ISSUE THREAD

**Issue by cp2587** [elastic/curator#861]
Hi,

Thank you for your work on this library. We recently updated to ES 5.0 and cannot use curator anymore because of the check that happens in 'verify_client_object', we subclassed elasticsearch.Elasticsearch and so the following line raise an exception:
```
elif not type(test) == type(elasticsearch.Elasticsearch()):  
     raise TypeError('Not a client object. Type: {0}'.format(type(test)))
```

Can i sugget you use something like this instead:
```
elif not isinstance(test, elasticsearch.Elasticsearch):  
     raise TypeError('Not a client object. Type: {0}'.format(type(test)))
```

**Comment by untergeek** [comment]
Feel free to submit a PR. It has to pass all tests. You might need to change one of the unit tests accordingly.

**Comment by cp2587** [comment]
Opened a PR: https://github.com/elastic/curator/pull/862 (i signed the CA after opening it, will it automatically update the checks ?)

**Comment by untergeek** [comment]
Closed by #862
